components/bl-static-breadcrumbs/src/index.js

```diff
@@ -0,0 +1,33 @@
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function StaticBreadcrumbsComponent({ component, eventHandlers, elRef }) {
+  const { classList, display, style, options } = component;
+  const { onItemClick, onMouseOver, onMouseOut } = eventHandlers;
+
+  const onClick = item => {
+    const { pageName, pageData } = item;
+
+    BackendlessUI.Navigator.goToPage(pageName, pageData);
+    onItemClick({ item });
+  };
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <div ref={ elRef } className={ cn('bl-customComponent-staticBreadcrumbs', classList) } style={ style }>
+      { options?.map(item => (
+        <div className="item">
+          <span
+            className="label"
```

Comment:
item and label are common classes, there could be a collisions.

Make it more specific